cpp/upgrades/81a312e6dafdffb650e1c7e3dc02cc37505e2505/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Differentiate between using aliases and typedefs
+compatibility: partial
```

Comment:
On upgraded dbs it will look like every typedef is C-style. What are the applications you have in mind for this feature, and how will they be affected?

Reply:
I don't think there are any specific plans to use it right now, it just seemed like something the AST ought to distinguish. It seems plausible that someone might want to distinguish them in a query, for example when codifying a style guide. I don't think we need to be concerned about upgraded databases, though.